### ISSUE TITLE
Users can see themselves in a list of people to add as friends

### ISSUE THREAD

**Issue by aozero** [CMPUT301F15T13/TheAllSwap#36]
As well as people they've already added as friends

**Comment by smithe0** [comment]
True. If you try to add yourself it yells at you but it's probably better you don't see it at all. This is the kind of thing we can fix for feedback though if we don't have time
